<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FtpOptDialog</class>
 <widget class="QWidget" name="FtpOptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="5">
    <widget class="QPushButton" name="btnCancel">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLineEdit" name="lEUser">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>anonymous</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="Label3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Download Offset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cBOffset">
     <property name="toolTip">
      <string/>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="Label7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="lbPassword">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QLineEdit" name="lEPassword">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="Label8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Time Offset in Path</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cBPathOffset">
     <property name="toolTip">
      <string/>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="Label9">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="Label4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Retry Interval</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QPushButton" name="btnOk">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="lbUser">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="Label2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Download Address (address/file-path)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnKey">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QComboBox" name="cBAddress">
     <property name="toolTip">
      <string/>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Label1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Download Interval</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cBInterval">
     <property name="toolTip">
      <string/>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="Label6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="cBRetryInterval">
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnKey</tabstop>
  <tabstop>cBAddress</tabstop>
  <tabstop>cBInterval</tabstop>
  <tabstop>lEUser</tabstop>
  <tabstop>cBOffset</tabstop>
  <tabstop>lEPassword</tabstop>
  <tabstop>cBPathOffset</tabstop>
  <tabstop>cBRetryInterval</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
